src/sessions/index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Adyen Checkout Components sample code</title>
+    <link rel="stylesheet" href="https://checkoutshopper-test.adyen.com/checkoutshopper/sdk/5.2.0/adyen.css">
+    <link rel="stylesheet" href="../demo.css">
+</head>
+<body>
+    <div class="container container--full-width">
+        <div class="main">
+            <div class="checkout-container">
+                <a href="../">Back</a>
+
+                <h1>Sessions Drop-in</h1>
+                <div class="payment-method">
+                    <div id="dropin-container">
+                        <!-- Drop-in will be rendered here -->
+                    </div>
+                    <div id="result-container"></div>
+                </div>
+            </div>
+
+            <div class="info">
+                <p>
+                    Check the Source Code to see the full implementation.
+                </p>
+                <p>
+                    To make a payment, use our <a href="https://docs.adyen.com/developers/development-resources/test-cards/test-card-numbers" target="_blank">test card numbers</a>.
+                </p>
+                <p>
+                    For more information, please refer to the <a href="https://docs.adyen.com/checkout/drop-in-web/" target="_blank">Drop-in documentation</a>.
+                </p>
+            </div>
+        </div>
+
+        <div class="sidebar">
+            <div class="header">
+                <h2>Basic Implementation</h2>
+                <button class="copy-sample-code" aria-label="Copy sample code"></button>
+            </div>
+            <pre class="source-code"><code>const checkout = await AdyenCheckout({
+    environment: 'test',
+    clientKey: 'test_...',
+    session: {
+        id: "CS62BB0....",
+        sessionData: "Ab02b4c0!BQABA....""
+    },
+    onPaymentCompleted: (result, component) => {
+        console.info(result, component);
+    },
+    onError: (error, component) => {
+        console.error(error.name, error.message, error.stack, component);
+    }
+});
+                            
+const dropin = checkout
+    .create('dropin')
+    .mount('#dropin-container');</code></pre>
+
+            <div class="header">
+                <h2>Current state</h2>
+            </div>
+            <pre class="current-state">{}</pre>
```

Comment:
@ribeiroguilherme - I fixed the first issue, calling the `onChange` / `onSelect`, the same as we do in the regular drop-in.

For the `Request` and `Response` container, I've changed it now so that `beforeSubmit` is called and updates the `Request` container, and `onPaymentCompleted` updates the `Response` container, and there's nothing before this (same as the regular drop-in).

Reply:
Looks good 💪